src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -182,6 +184,31 @@ DlgPrefInterface::DlgPrefInterface(
     int inhibitsettings = static_cast<int>(m_pScreensaverManager->status());
     comboBoxScreensaver->setCurrentIndex(comboBoxScreensaver->findData(inhibitsettings));
 
+    // Multi-Sampling
+#ifdef MIXXX_USE_QML
+    if (CmdlineArgs::Instance().isQml()) {
+        mulitSamplingComboBox->clear();
```

Comment:
Typo

Reply:
damn... why did CI not catch this?

Reply:
Codespell uses a dictionary of common misspellings and does not support camelcase at the moment. The chance that "mulitsamplingcombobox" is in the dict is rather small :)